qa/integration/services/kafka_setup.sh

```diff
@@ -23,7 +23,7 @@ setup_kafka() {
     local version=$1
     if [ ! -d $KAFKA_HOME ]; then
         echo "Downloading Kafka version $version"
-        curl -s -o $INSTALL_DIR/kafka.tgz "https://mirrors.ocf.berkeley.edu/apache/kafka/$version/kafka_2.11-$version.tgz"
+        curl -s -o $INSTALL_DIR/kafka.tgz "https://ftp.wayne.edu/apache/kafka/$version/kafka_2.13-$version.tgz"
```

Comment:
Is there a reason we don't use apache's download service directly? We've been hit by mirrors disappearing many times.

```suggestion
        curl -s -o $INSTALL_DIR/kafka.tgz "https://downloads.apache.org/kafka/$version/kafka_2.13-$version.tgz"
```